## The chromatin immunoprecipitation (ChIP) assay


### ChIP-Sequencing

Chromatin Immuno-precipitation and Sequencing (ChIP-Sequencing)[[1](#references)] is widely used in the study of DNA-protein interactions. This method is used to identify the DNA binding targets of transcription factors across the genome, providing the researcher with a catalogue of DNA elements regulated by the protein of interest. Histone modifications are also widely assayed using this technique, and give insight into the structure and activation status of nearby genes. Many variations of this technique are also available, and may be more suitable depending on the research question at hand and material availability.

ChIP-Sequencing has provided landmark insights in genome biology and gene regulatory understanding. A large database of ChIP-Sequencing experiments is available through the ENCODE project. Dr. Barski’s longstanding interest in methods development and DNA-protein interactions has yielded numerous publications utilizing this technology[[2–5](#references)].

### Method:

ChIP-Sequencing begins with the harvesting and fixation of the cell type of interest. Fixation using formaldehyde will cross-link the DNA with histones and trans-factors, capturing the complexes and conformations in one moment of time. Samples are then sonicated to shear DNA fragments to be immunoprecipitated (commonly utilizing a Covaris sonicator). Careful titration of fixation times and the sonication will vary per cell type. Samples may be manually Immunoprecipitated, or handled using one of the many automated robotics systems overnight. Both procedures will enrich your molecule of interest using antibody pulldown with protein A/G. Samples are de-crosslinked to break free the DNA from the protein of interest, and amplified using a library preparation kit for NGS sequencing. Recent techniques allow for enhanced target binding sensitivity using ChIPMentation for library preparation6. Samples with compatible indexes may be pooled for NGS sequencing. Biowardrobe will handle the .fastq file which most DNA cores and commercial NGS companies return, and prepare you for data analysis.


### ChIP-Seq analysis in SciDAP

[![ChIP-Seq](http://img.youtube.com/vi/6Fs3xb9fXII/0.jpg)](http://www.youtube.com/watch?v=6Fs3xb9fXII "ChIP-Seq analysis in SciDAP")


### View ChIP-Seq results in SciDAP

[![ChIP-Seq](http://img.youtube.com/vi/K7b7ieJlol4/0.jpg)](http://www.youtube.com/watch?v=K7b7ieJlol4 "View ChIP-Seq results in SciDAP")

### References

1. Barski A, Cuddapah S, Cui K, Roh T-Y, Schones DE, Wang Z, Wei G, Chepelev I, Zhao K. High-resolution profiling of histone methylations in the human genome Cell. 2007 May;129(4):823–37. PMID: 17512414
2. Yukawa M, Jagannathan S, Vallabh S, Kartashov A V, Chen X, Weirauch MT, Barski A. AP-1 activity induced by co-stimulation is required for chromatin opening during T cell activation. J Exp Med [Internet]. 2019 Oct 25;jem.20182009. Available from: http://jem.rupress.org/content/early/2019/10/24/jem.20182009.abstract
3. Barski A. AP-1 transcription factor reprograms T cell epigenome during activation. J Immunol [Internet]. 2017 May 1;198(1 Supplement):124.6 LP-124.6. Available from: http://www.jimmunol.org/content/198/1_Supplement/124.6.abstract
4. Wang Z, Zang C, Cui K, Schones DE, Barski A, Peng W, Zhao K. Genome-wide mapping of HATs and HDACs reveals distinct functions in active and inactive genes. Cell. United States; 2009 Sep;138(5):1019–1031. PMID: 19698979
5. Barski A, Jothi R, Cuddapah S, Cui K, Roh T-Y, Schones DE, Zhao K. Chromatin poises miRNA- and protein-coding genes for expression. Genome Res. 2009;19(10).
